src/index.html

```diff
@@ -2,12 +2,52 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card-imac-image"
+        src="images/imac.jpeg"
+        alt="iMac A1419"
+      >
+
+      <h1 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <h2 class="card__product-code">Product code: 195434</h2>
+
+      <div class="card__rating-info">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__reviews-5">Reviews: 5</div>
+      </div>
+
+      <div class="card__price-container">
+        <div class="card__price-label">Price:</div>
+        <div class="card__price">$2,199</div>
```

Comment:
Still not fixed from the previous review. You need to use a semantic tag everywhere for text 
<img width="765" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/b8c7b37a-b2d3-489d-96e6-02d63ba44ba4">